src/WebJobs.Script.WebHost/Security/HostSecrets.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class HostSecrets
+    {
+        [JsonProperty(PropertyName = "masterKey")]
+        public Key MasterKey { get; set; }
+
+        [JsonProperty(PropertyName = "functionKeys")]
+        public IList<Key> FunctionKeys { get; set; }
+
+        [JsonProperty(PropertyName = "version")]
+        private int Version
```

Comment:
I might have asked this before, but I'm wondering if we really need a Version property. Note that none of our other metadata files have a versioning property/strategy.

If this is just to aid this one time format change during preview, can't you just "sniff" for the two new properties to determine whether you're dealing with the old or new format? That way we don't need a versioning infrastructure for this.